.stylelintrc.json

```diff
@@ -14,6 +15,7 @@
     "unit-no-unknown": true,
     "block-no-empty": true,
     "declaration-block-no-duplicate-properties": true,
-    "number-leading-zero": "always"
+    "number-leading-zero": "always",
+    "no-global-function-names": "never"
```

Comment:
Did I add these? I think the no-global-function-names is a good one to have in there